jobsAdmin/init.go

```diff
@@ -431,13 +434,31 @@ func GetJobSummary(jobID common.JobID) common.ListJobSummaryResponse {
 	part0PlanStatus := part0.Plan().JobStatus()
 
 	// Add on byte count from files in flight, to get a more accurate running total
-	js.TotalBytesTransferred += jm.SuccessfulBytesInActiveFiles()
+	// Check is added to prevent double counting
+	if js.TotalBytesTransferred+jm.SuccessfulBytesInActiveFiles() <= js.TotalBytesExpected {
```

Comment:
This could still look wrong but I guess it's "less" wrong? Been scratching my head on a more accurate but equally efficient algorithm, and I don't have a good answer.